components/src/dynamo/vllm/instrumented_scheduler.py

```
@@ -2577,6 +2577,16 @@ def _bench_prefill_point_feasible(
         except ValueError:
             return False
 
+        eagle_cache_drop_tokens = self._bench_eagle_cache_drop_tokens()
+        if eagle_cache_drop_tokens and any(
+            kv_read_tokens > 0 and new_tokens <= eagle_cache_drop_tokens
+            for new_tokens, kv_read_tokens in zip(new_token_lengths, kv_read_lengths)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)